catalog/core/catalog-core-api-impl/src/test/java/ddf/catalog/endpoint/impl/CatalogEndpointImplTest.java

```diff
@@ -0,0 +1,143 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package ddf.catalog.endpoint.impl;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Test;
+
+public class CatalogEndpointImplTest {
+
+    private static final String DEFAULT_BINDING_TYPE = "defaultBindingType";
+
+    private static final String DEFAULT_DESCRIPTION = "description";
+
+    private static final String DEFAULT_ID = "id";
+
+    private static final String DEFAULT_NAME = "name";
+
+    private static final String DEFAULT_URL = "url";
+
+    private static final String DEFAULT_URL_BINDING_NAME = "urlBindingName";
+
+    private static final String DEFAULT_VERSION = "version";
+
+    @Test
+    public void testSettingEndpointProperties() {
+        Map<String, String> propertiesMap = getTestEndpointPropertiesWithDefaults();
+
+        CatalogEndpointImpl catalogEndpoint = new CatalogEndpointImpl();
+        Map<String, String> endpointProperties = catalogEndpoint.getEndpointProperties();
+        assertTrue("Endpoint Properties Empty? Should be.", endpointProperties.isEmpty());
```

Comment:
These messages are a little funny.   I'd phrase it more like: `"Endpoint Properties weren't empty."

Similar for the other messages in this file.